### PR TITLE
Add python azure.identity.extentions code

### PR DESCRIPTION
**Pull request by v-xuto**
**1. DefaultAzureMgmtCredential**  
The DefaultAzureMgmtCredential class provides a simple bridge to use DefaultAzureCredential from Azure.Identity SDKs. 
**2. DefaultAzureServiceBusCredential**
The DefaultAzureServiceBusCredential class (based on Azure.Identity.DefaultAzureCredential) now can't implement for the following reasons:
The user authentication credentials of the TopicClient sending messages in the servicebus depend on the two parameters shared_access_key_name and shared_access_key_value in the parent mixins.baseclient constructor. This is the only way for certification, and this way can't replace with azure. Identity. DefaultAzureCredential created the credential. In c#, there are several different ways to authenticate, including the TopicClient accepting an ItokenProvider parameter, but there is no such method in python.

### CODE REVIEW

**Comment by v-xuto** [comment]
For python, I try to pass DefaultAzureMgmtCredential, it is invalid for TopicClient.
The `TopicClient` base class `BaseClient `does not provide a method to pass credential. As shown in the following figure:
image

**Comment by jongio** [comment]
Let's focus on getting DefaultAzureMgmtCredential done.

File: README.md [hunk 1]
```diff
@@ -114,4 +114,29 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
 
 Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
+## Python
+
+### default_azure_mgmt_credential.py
+
+The `DefaultAzureMgmtCredential` class provides a simple bridge to use `DefaultAzureCredential` from `Azure.Identity` SDKs. This is a convenient mechanism to authenticate all Azure Management Resources and a some data plane SDKs that use `ServiceClientCredential` family of credentials.
```
Comment (jongio) [inline README.md:121]: Just this for this paragraph: The `DefaultAzureMgmtCredential` class provides a simple bridge to use `DefaultAzureCredential` from `Azure.Identity` SDKs with the Azure Management Plane SDKs.

File: README.md [hunk 2]
```diff
@@ -114,4 +114,29 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
...
+
+To use this type, just copy `default_azure_mgmt_credential.py` file located in `python`directory into your application and make necessary package name updates.
```
Comment (jongio) [inline README.md:123]: Space after `python`

File: README.md [hunk 3]
```diff
@@ -114,4 +114,29 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
...
+client = ServiceManagementClient(credential, subscription_id)
+```
+
+#### Testing DefaultAzureMgmtCredential
+
+This repository has a test class called `TestDefaultAzureMgmtCredential` that tests creation of a storage account, listing all storage accounts in a resource group to validate successful creation, then deleting the account created earlier in this test and listing again to ensure successful deletion.
+
+To run `TestDefaultAzureMgmtCredential`, please ensure that you have already set up the following environment variables:
+
+- AZURE_SUBSCRIPTION_ID
```
Comment (jongio) [inline README.md:138]: Please use .env to load and python-dotenv project to load.

Comment (jongio) [reply README.md:138]: We already have a .env in the root.  Thanks

File: python/default_azure_mgmt_credential.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+from azure.common.credentials import ServicePrincipalCredentials
+import os
+import time
+
+
+class DefaultAzureMgmtCredential(ServicePrincipalCredentials):
+    scopes = 'https://management.azure.com/.default'
+    accessTokenCache = {}
+
+    def __init__(self, client_id=None, secret=None, credentail=None, **kwargs):
```
Comment (lmazuel) [inline python/default_azure_mgmt_credential.py:11]: - You shouldn't expose client_id/secret, it brings no value
- credentail is a typo
- scopes should be available as kwargs to override

**Comment by jongio** [comment]
This was implemented here: https://github.com/jongio/azidext/pull/12

**Pull request closed without merging**